src/Microsoft.ML.Recommender/Microsoft.ML.Recommender.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
```

Comment:
What NuGet package is this going to be a part of?  I assume it won't be the main `Microsoft.ML` nuget package because of the extra dependency on `libmf`.

Reply:
Are you saying that LIBMFNative should be a separated nuget?